flight/PiOS/STM32F10x/pios_reset.c

```diff
@@ -0,0 +1,48 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_RESET Reset functions
+ * @brief Hardware functions to deal with the reset register
+ * @{
+ *
+ * @file       pios_reset.c
+ * @author     Kenn Sebesta, Copyright (C) 2015.
```

Comment:
All these need to be updated. They're written with internal copyrights so that the git history makes it clear that code has been deliberately open-sourced. Thanks for catching this, I'll try to do better in the future.